### DISCUSSION TITLE
Command for global chat

### DISCUSSION THREAD

**Discussion by Holwens** [sinipelto/dayz-scripts#1] [Ideas]
Hello.

First of all thanks for your work.

I am with the DayZ Script, and I have no way to do it since my programming knowledge is low.

Is there a way to isolate the /say command from admin permissions?

And every time the command is written, does the player's name appear plus the text that he writes at the beginning of the chat sentence?

Thanks in advance.

**Comment by sinipelto** [comment]
Hi!
Thank you for the suggestion! Currently, all commands are for admins only, a non-admin person cannot execute any of the commands if not his/her SteamID put in the admins list text file.  Currently, only the message appears in the chat but this could be adjusted or split into a separate commands which would include the announcing player name. Also, that is a great suggestion, it should be relatively straightforward to modify the logic so that you could define which commands are for admins only and which are for everyone.
